Setting WITH_LUAJIT results in error

When you try to compile `wrk` by setting the parameter `WITH_LUAJIT` it results in following error
```
$ make WITH_LUAJIT=/usr WITH_OPENSSL=/usr

CC src/wrk.c
In file included from src/wrk.c:3:0:
src/wrk.h:13:17: fatal error: lua.h: No such file or directory
 #include <lua.h>
                 ^
compilation terminated.
make: *** [Makefile:71: obj/wrk.o] Error 1
```
I also added the `WITH_OPENSSL` parameter, so it doesn't compile openssl at the beginning.

--- a/Makefile
+++ b/Makefile
@@ -30,11 +30,13 @@ DEPS    :=
 CFLAGS  += -I$(ODIR)/include
 LDFLAGS += -L$(ODIR)/lib
 
+LUAJIT_VER ?= luajit-2.1
+
 ifneq ($(WITH_LUAJIT),)
-	CFLAGS  += -I$(WITH_LUAJIT)/include
+	CFLAGS  += -I$(WITH_LUAJIT)/include/$(LUAJIT_VER)
 	LDFLAGS += -L$(WITH_LUAJIT)/lib
 else
-	CFLAGS  += -I$(ODIR)/include/luajit-2.1
+	CFLAGS  += -I$(ODIR)/include/$(LUAJIT_VER)
 	DEPS    += $(ODIR)/lib/libluajit-5.1.a
 endif
 

